tests/wamr-test-suites/test_wamr.sh

```diff
@@ -469,6 +469,37 @@ function wasi_test()
     echo "Finish wasi tests"
 }
 
+function wasi_test()
+{
+    echo  "Now start wasi tests"
+    
+    cd ${WORK_DIR}
+    if [ ! -d "wasi-testsuite" ]; then
+        echo "wasi not exist, clone it from github"
+        git clone -b prod/testsuite-base \
+            --single-branch https://github.com/WebAssembly/wasi-testsuite.git
+    else
+        echo "wasi exists, pulling updates from github"
+        pushd wasi-testsuite
+        git fetch
+        git reset --hard origin/prod/testsuite-base
+        popd
+    fi
+
+    cd wasi-testsuite
+    python3 -m pip install -r test-runner/requirements.txt
```

Comment:
I wonder if we can do it in virtual environment [1] so we don't pollute host's environment with packages that they might not want to have? 

[1] https://docs.python.org/3/tutorial/venv.html